src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -15,5 +15,7 @@
     public static final String MESSAGE_TASKS_LISTED_OVERVIEW = "%1$d task(s) listed!";
     public static final String MESSAGE_END_DATETIME_BEFORE_START_DATETIME = "The event end date and time "
             + "is before start date and time!";
+    public static final String MESSAGE_INVALID_DATE = "The date format is invalid! "
+            + "Date should be in the format of uuuu-MM-DD!";
```

Comment:
I think for user, "YYYY-MM-DD" would be better, as "uuuu-MM-DD" is lower level stuff.
(Side info: "uuuu-MM-DD" can deal with leap year 0229 problem, generally speaking, it's just a strict version of "yyyy-MM-DD")